linode/manager
# feat: [M3-6816] - Add AGLB Create Route Drawer

## Description 📝
Implements the AGLB create route drawer.

## Preview 📷
| Drawer      | Error |
| ----------- | ----------- |
|![Screenshot 2023-10-17 at 1 24 39 PM](https://github.com/linode/manager/assets/125309814/f0b22c88-3648-462e-9e2e-4d93d2c1bf12)|![Screenshot 2023-10-17 at 1 24 49 PM](https://github.com/linode/manager/assets/125309814/7fb170c0-55ea-48d8-87cd-eb91b40906fd) |


## How to test 🧪

### Prerequisites
- Checkout this PR
- Turn on the MSW (bottom left dev tools)

### Reproduction steps
- Go to http://localhost:3000/loadbalancers/0/routes
- Click `Create Route`

### Verification steps 
- Observe drawer opens with label input field and protocol options.
- Observe when click `Create Route` _without_ a label, an error is surfaced.
- Run cypress tests `yarn cy:run -s cypress/e2e/core/loadBalancers/load-balancer-routes.spec.ts`

## As an Author I have considered 🤔

*Check all that apply*
